fggs/domains.py

```diff
@@ -55,3 +55,33 @@ def __eq__(self, other):
 
     def __ne__(self, other):
         return not self.__eq__(other)
+
+
+class CompactDomain(Domain):
+    """A compact domain for finite sets (like vocabularies).
+
+    Compact domain is like finite domain, but it doesn't hold
+    vocabularies. Instead it only holds the size of values.
```

Comment:
This line is confusing; the first line said that CompactDomain is for vocabularies, but this line says that it doesn't hold vocabularies.

I think it would be better to say that it stores the integers 0, ..., size-1 but doesn't store any names.

Reply:
This comment has been changed to make it clearer.